policyengine_us/parameters/gov/states/dc/dhcf/medicaid/eligibility/adult_income_limit.yaml

```
@@ -0,0 +1,16 @@
+description: DC limits Medicaid/Alliance for adults (21+) to this percentage of the Federal Poverty Level
```

```suggestion
description: The District of Columbia limits Medicaid/Alliance for adults (21+) to this percentage of the Federal Poverty Level.
```
And other description in parameter files.